Formula/ripasso-cursive.rb

```diff
@@ -0,0 +1,37 @@
+class RipassoCursive < Formula
+  desc "Tui password manager compatible with 'pass', written in rust"
+  homepage "https://github.com/cortex/ripasso"
+  url "https://github.com/cortex/ripasso/archive/refs/tags/release-0.6.3.tar.gz"
+  sha256 "2f43e48c3e1064bef47c89269e4797f216b7600b850f3857cfce7e9716ef8f7d"
+  license "GPL-3.0-only"
+
+  depends_on "libxcb" => :build
+  depends_on "llvm" => :build
```

Comment:
Why is `llvm` needed?

Reply:
The nettle crypto library nettle depends on libclang.so, and that shared library is provided by llvm.

Without it the build fails with this (abbreviated):
```
error: failed to run custom build command for `nettle-sys v2.1.1`

Caused by:
  process didn't exit successfully: `/tmp/ripasso-cursive-20230228-276307-itsb6n/ripasso-release-0.6.3/target/release/build/nettle-sys-0ae80325189241dc/build-script-build` (exit status: 101)
...
  thread 'main' panicked at 'Unable to find libclang: "couldn't find any valid shared libraries matching: ['libclang.so', 'libclang-*.so', 'libclang.so.*', 'libclang-*.so.*'], set the `LIBCLANG_PATH` environment variable to a path where one of these files can be found (invalid: [])"', /home/capitol/.cache/Homebrew/cargo_cache/registry/src/github.com-1ecc6299db9ec823/bindgen-0.63.0/./lib.rs:2338:31
```

Reply:
It should use the nettle formula then

Reply:
It is using the nettle formula

Reply:
I was maybe a bit technically incorrect, it's the rust crate `nettle-sys` that depends on libclang, I don't know if the nettle shared object also depends on libclang. I suspect that it might not do that.

Reply:
@alexanderkjall can you also try `uses_from_macos "llvm" # for libclang` as it is used by some other formulae as well.